### ISSUE TITLE
Unable to run with LocalDB

### ISSUE THREAD

**Issue by trailmax** [SharpeRAD/Cake.SqlTools#1]
I'm using VSTS and the only database available is LocalDB, but `SqlClientLibrary` apparently can't work with LocalDB on attempt to open connection I get network error:  

> A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)

The instance is up and running - I'm using it from other applications. 
Here is how build.cake looks like:

    Task("Create-DB-And-Schema")
    .Does(() => 
    {
        ExecuteSqlQuery("create database [MyProject-Testing]", new SqlQuerySettings()
        {
            Provider = "MsSql",
            ConnectionString = "Server=(localdb)\v12.0;"
        });
    })

Here is a repro:

    class Program
    {
        static void Main(string[] args)
        {
            System.Data.IDbConnection con = System.Data.SqlClient.SqlClientFactory.Instance.CreateConnection();

            con.ConnectionString = "Server=(localdb)\v12.0;";
            con.Open(); // throws here
        }
    }

**Comment by SharpeRAD** [comment]
You problem is with SqlClientLibrary and not the addin itself. You can support multiple databases by implementing a new [ISqlQueryRepository](https://github.com/SharpeRAD/Cake.SqlTools/blob/master/src/SqlTools/Interfaces/ISqlQueryRepository.cs) and updating the Alias with the new [provider](https://github.com/SharpeRAD/Cake.SqlTools/blob/master/src/SqlTools/Aliases/SqlQueryAliases.cs#L42)

Happy to accept a PR for this when you find a solution :wink:

**Comment by trailmax** [comment]
Yeah.. after enough head banging I found the issue. Two words "String escaping".

The connection string I used in Cake was `var connStr = "Server=(localdb)\v12.0;";`. But Cake is C#. And in C# backwards slash `\` is an escape symbol, so it really should be `var connStr = @"Server=(localdb)\v12.0;";`

So actually it was not SqlClientLibrary's fault. It was my fault for not putting `@` in front of the connection string. Sigh. Simple thing are always the worst to figure out!
